ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/underlay-overlay/super-spine-overlay-loopback.j2

```diff
@@ -0,0 +1,4 @@
+{# Super Spine Loopback Interface for Overlay Peering #}
+  Loopback0:
+    description: EVPN_Overlay_Peering
```

Comment:
Maybe we should reconsider the description of loopback0. EVPN_Overlay_Peering will not make sense unless the super_spine is also Route-Server for EVPN.
I still like to keep the loopback.

Reply:
I agree with @ClausHolbechArista - maybe just have a conditional on the description based on how they are used? This would also apply to spine loopback0.